libs/external-router/src/external-router.module.ts

```diff
@@ -22,6 +23,7 @@ import { provideRoutesResolvableByType } from './token/type-resolvable-routes.to
 export class DaffExternalRouterModule {
   static forRoot(
     routes: TypeRoutePair[] = [],
+    inserters: DaffExternalRouterTypeRouteInserterPair[] = [],
```

Comment:
Can we instead expand the TypeRoutePair with an optional

```ts
{
 ...
 insertionStrategy?: DaffExternalRouterRouteInserter
 }
```

Reply:
That would force the app dev to provide the insertion strategy when they provide the route type. Is that what we want?

Reply:
Note the optional, in the event the prop isn't defined, we use the afterWildcard strategy.

Reply:
yes but if they do want to provide a strategy, it must be when they provide the route type.

Reply:
I suppose its likely that they would be able to provide the strategy at that point. As soon as you know the route type you most likely know how to insert it into your route config.

Reply:
I suspect so, and it keeps the API much more simple.

Reply:
changed